src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebReportRouteConfig.kt

```diff
@@ -76,6 +76,12 @@ object WebReportRouteConfig : RouteConfig
             WebEndpoint("/report/:name/dependencies/",
                     ReportController::class, "getDependencies")
                     .json()
-                    .secure(readReports)
+                    .secure(readReports),
+            WebEndpoint(
+                    "/view-workflow/",
```

Comment:
I suggest `/workflows/:key` - `key` is not optional

Reply:
I think with reports etc we've tended to use singular when we retrieve details of one thing, and plural to get a list, so do you think `/workflow/:key` would be more consistent?

Reply:
I think we have some inconsistency... plurals are used in [ReportRouteConfig](https://github.com/vimc/orderly-web/blob/master/src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/api/ReportRouteConfig.kt). I think it's preferable to have related routes under a common prefix - certainly for JSON endpoints - and broader convention seems to follow this pattern (Stripe API being the gold standard).

Reply:
I think `/workflows/:key/ ` should be preferable. It also looks like your ticket mrc-2305 has implemented webWorkflowRouteConfig so I should move this route as well.